tests/fixtures/custom-schemas/invalid/schema-1.yml

```diff
@@ -0,0 +1,14 @@
+schema: Custom.SampleSchema1
```

Comment:
I fail to see.. why is this invalid?

Reply:
It's not actually, I will add a comment to clarify. The point was to mix valid and invalid schemas together and test the behavior.